common/src/common/misc/SpecialFunctions.cpp

```diff
@@ -495,3 +497,22 @@ void SpecialFunctions::spawnCrystal(Functional* obj, const j3x::Obj& data, Chara
     auto pos = j3x::getObj<sf::Vector2f>(data);
     framework_->spawnSpecial(pos, "crystal");
 }
+
+void SpecialFunctions::spawnAnimationEvent(Functional* obj, const j3x::Obj& data, Character* user)
+{
+    LOG.info("[SpecialFunction] Spawning animation.");
+    auto& data_list = j3x::getObj<j3x::List>(data);
+    const auto& animation = j3x::getObj<std::string>(data_list, 0);
+    const auto& pos = j3x::getObj<sf::Vector2f>(data_list, 1);
+    auto number = std::to_string(utils::num::getRandom(1, 1));
```

Comment:
this should be removed very long time ago! just leave "animation" variable. is this even working?